source/stylesheets/refills/_footer.scss

```diff
@@ -64,7 +64,7 @@
   li h3 {
     color: $footer-color;
     font-size: 1em;
-    font-weight: 800;
+    font-weight: 700;
```

Comment:
this might be worth a variable so that next time you need to change it it's only in one place